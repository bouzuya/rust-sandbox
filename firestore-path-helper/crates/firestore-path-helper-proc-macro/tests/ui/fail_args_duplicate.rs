fn main() {
    firestore_path_helper_proc_macro::firestore_path_helper!(
        "users/{user_id}",
        user_id = String,
        user_id = String, // duplicate argument
    );
}
