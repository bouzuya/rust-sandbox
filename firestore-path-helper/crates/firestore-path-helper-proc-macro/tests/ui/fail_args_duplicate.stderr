error: duplicate document id argument: 'user_id'
 --> tests/ui/fail_args_duplicate.rs:5:9
  |
5 |         user_id = String, // duplicate argument
  |         ^^^^^^^
